llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -15238,16 +15238,18 @@ static SDValue isScalarToVec(SDValue Op) {
 // On little endian, that's just the corresponding element in the other
 // half of the vector. On big endian, it is in the same half but right
 // justified rather than left justified in that half.
-static void fixupShuffleMaskForPermutedSToV(SmallVectorImpl<int> &ShuffV,
-                                            int LHSMaxIdx, int RHSMinIdx,
-                                            int RHSMaxIdx, int HalfVec,
-                                            unsigned ValidLaneWidth,
-                                            const PPCSubtarget &Subtarget) {
+static void fixupShuffleMaskForPermutedSToV(
+    SmallVectorImpl<int> &ShuffV, int LHSFirstElt, int LHSLastElt,
+    int RHSFirstElt, int RHSLastElt, int HalfVec, unsigned LHSNumValidElts,
+    unsigned RHSNumValidElts, const PPCSubtarget &Subtarget) {
   for (int i = 0, e = ShuffV.size(); i < e; i++) {
```

Comment:
Sorry,  a nit: according to the coding standard, [variable names should be uppercase](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly). In addition, [preincrement is preferred](https://llvm.org/docs/CodingStandards.html#prefer-preincrement):
 ```suggestion
  for (int I = 0, E = ShuffV.size(); I < E; ++I) {
```